### ISSUE TITLE
BottomSheet rotation issue

### ISSUE THREAD

**Issue by kmaslowiec** [Android-Guild/Picover#276]
- [ ] Opened BottomSheet when the rotation is from vertical to horizontal crushes the app and causes: java.lang.IllegalStateException: The offset was read before being initialized. Please see: https://github.com/google/accompanist/issues/1493 

https://github.com/intive/Picover/assets/33199010/d4ba6889-2c2f-4137-9a6c-2c686eda3587

Possible solution: upgrade Material Design to [1.2.0-alpha08]

- [x] The BottomSheet is being hidden during the screen rotation #354 

https://github.com/intive/Picover/assets/33199010/0dc9a1dc-86da-4b37-b40c-3300e4b6194e

**Comment by michal-kucznerowicz** [comment]
Decided to wait for a stable version with a fix.

**Comment by kmaslowiec** [comment]
The issue appears only in our project. Even though the second issue can be fixed with the upgraded Material3, it can be fixed differently. It needs to be addressed.

**Comment by kmaslowiec** [comment]
When we change orientation we are using two different NavHosts. One in PicoverNavigationRail (horizontal) and the other PicoverNavigationBar. 
Hypothesis: Implementing one nav host for both components can resolve the issue.

**Comment by kmaslowiec** [comment]
Maybe rememberUpdatedState? Needs to be investigated.

**Comment by kmaslowiec** [comment]
Source:
Official: https://google.github.io/accompanist/
FAQ: https://medium.com/androiddevelopers/jetpack-compose-accompanist-an-faq-b55117b02712
GitHub: https://github.com/google/accompanist

**Comment by MarekMacko** [comment]
@kmaslowiec Is there anything do to left?

**Comment by kmaslowiec** [comment]
> @kmaslowiec Is there anything do to left?

@MarekMacko Yep, we still need to wait for the stable version that will fix the crash. Please see above: [up](https://github.com/Android-Guild/Picover/issues/276#issue-1917083139)
